egs2/qasr_tts/tts1/conf/tuning/train_conformer_fastspeech.yaml

```diff
@@ -0,0 +1,88 @@
+# This configuration is for ESPnet2 to train Conformer-based
```

Comment:
Is there a need to keep the fast speech config?

Reply:
Yes